doc/BUILDING.rst

```diff
@@ -188,8 +188,10 @@ RPi3 ARMv7 and LePotato ARM64
 
 We support cross-compilation from Linux hosts. The following ``--config`` flags can be specified when building with bazel:
 
-* ``--config=rpi3_opt`` for Raspbian / ARMv7
-* ``--config=rpi3-armv8_opt`` for ARMBian / ARM64
+* ``--config=elinux_armhf`` for Raspbian / ARMv7
+* ``--config=elinux_aarch64`` for ARMBian / ARM64
+
+Checkout `Tensorflow's guide on cross-compilation for ARM with Bazel <https://www.tensorflow.org/lite/guide/build_arm#cross-compilation_for_arm_with_bazel>`_
```

Comment:
Upstream links tends to be often broken, and the lines after are alread good IMHO

Reply:
(except they reference `rpi3_opt` and `rpi3-armv8_opt` so you probably want to update that)

Reply:
I understand.